js/client-chat.js

```diff
@@ -186,6 +186,8 @@
 
 		getHighlight: function (message) {
 			var highlights = Tools.prefs('highlights') || [];
+			var roomHighlights = (this.type === 'battle' ? Tools.prefs('battlehls') || [] : Tools.prefs(this.id) || []);
```

Comment:
would it be better if i did it as room.id + 'hls' ? ex: lobby would be under lobbyhls